src/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -0,0 +1,29 @@
+import { rules } from "../rule";
+
+rules.push({
+  id: "M2065",
+  name: "DescriptionMustNotBeNodeName",
+  severity: "error",
+  category: ["RPCViolation"],
+
+  appliesTo_JsonQuery: "$..*[?(@.description)]",
+  run: function* (doc, node, path) {
+    let msg: string = "Description must not match the name of the node it is supposed to describe";
```

Comment:
I'm in the process of adding the other 2 validation categories and the mechanism to filter based on the categorizations.